plugins/org.jkiss.dbeaver.erd.model/src/org/jkiss/dbeaver/erd/model/ERDAttributeVisibility.java

```diff
@@ -59,6 +59,11 @@ public static ERDAttributeVisibility getDefaultVisibility(DBPPreferenceStore sto
         return ALL;
     }
 
+    public static boolean isHideAttributeAssociations(DBPPreferenceStore store) {
+        ERDAttributeVisibility defaultVisibility = getDefaultVisibility(store);
+        return defaultVisibility != ALL;
```

Comment:
```suggestion
        return getDefaultVisibility(store) != ALL;
```

I think this method name is misleading. At least I find it confusing. Perhaps it can be inlined or called more appropriately?